src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -853,6 +854,29 @@ double christodoulou_mass(const double dimensionful_spin_magnitude,
   return sqrt(square(irreducible_mass) + (square(dimensionful_spin_magnitude) /
                                           (4.0 * square(irreducible_mass))));
 }
+
+template <typename Frame>
+void radial_distance(gsl::not_null<Scalar<DataVector>*> radial_distance,
+                     const Strahlkorper<Frame>& strahlkorper_a,
+                     const Strahlkorper<Frame>& strahlkorper_b) {
+  if (strahlkorper_a.center() != strahlkorper_b.center()) {
+    ERROR(
+        "Currently computing the radial distance between two Strahlkorpers "
+        "is only supported if they have the same centers, but the "
+        "strahlkorpers provided have centers "
+        << strahlkorper_a.center() << " and " << strahlkorper_b.center());
+  }
+  get(*radial_distance)
+      .destructive_resize(strahlkorper_a.ylm_spherepack().physical_size());
+  get(*radial_distance) =
+      (strahlkorper_a.l_max() == strahlkorper_b.l_max() and
+       strahlkorper_a.m_max() == strahlkorper_b.m_max())
+          ? abs(strahlkorper_a.radius() - strahlkorper_b.radius())
+          : abs(strahlkorper_a.radius() -
+                Strahlkorper<Frame>(strahlkorper_a.l_max(),
+                                    strahlkorper_a.m_max(), strahlkorper_b)
```

Comment:
Does it matter if `strahlkorper_a.l_max()`  is greater than `strahlkorper_b.l_max()`?  The docs for this function imply that `strahlkorper_a.l_max()` must be less than or equal to `strahlkorper_b.l_max()`. (same for `m_max()`)

Reply:
Clarified the docs that I'm constructing a new strahlkorper with the constructor that prolongs or restricts the extents to match the desired l_max, m_max